topics/reference_guide/custom_language_support/websymbols.md

```diff
@@ -0,0 +1,31 @@
+<!-- Copyright 2000-2023 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file. -->
```

Comment:
```suggestion
<!-- Copyright 2000-2023 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license. -->

```
Please also notice a line break after copyright.

Reply:
How on earth did you notice this? :)

It was of course c&p from another file..